devtools/runproc/signals.go

```diff
@@ -0,0 +1,5 @@
+package main
```

Comment:
platform differences. `signals.go` builds on all platforms and has the base declarations, `signals_unix.go` only builds on unix-systems and includes `unix.SIGTERM` which isn't defined on things like windows (and would fail to compile otherwise)